Client/Frontend/ContentBlocker/ContentBlockerSettingViewController.swift

```diff
@@ -61,7 +61,8 @@ class ContentBlockerSettingViewController: ContentBlockerSettingsTableView {
 
     override func generateSettings() -> [SettingSection] {
         let enabledSetting: [CheckmarkSetting] = EnabledStates.map { option in
-            return CheckmarkSetting(title: NSAttributedString(string: option.settingTitle), subtitle: nil, isEnabled: {
+            let id = ContentBlockerHelper.EnabledState.accessibilityId(for: option)
+            return CheckmarkSetting(title: NSAttributedString(string: option.settingTitle), subtitle: nil, accessibilityIdentifier: id, isEnabled: {
```

Comment:
In order for the screengraph to change the Settings, I needed proper  accessibilityIdentifier set on the options (I wasn't aware when I landed this a while ago of the implications of not doing that, now I know, do it the first time).